lxd/instance/instance_utils.go

```diff
@@ -693,6 +693,12 @@ func CreateInternal(s *state.State, args db.InstanceArgs, clearLogDir bool) (Ins
 
 	args.Config["volatile.uuid.generation"] = args.Config["volatile.uuid"]
 
+	// Validate pinning strategy when limits.cpu specifies static pinning
+	cpuPinningSpecified := strings.ContainsAny(args.Config["limits.cpu"], ",-")
```

Comment:
This validation shouldnt be here

Reply:
I've moved this input validation to `ValidConfig()`.

Reply:
Let me know if a validation function in `InstanceConfigKeysVM` would be preferred for this.